### ISSUE TITLE
REALLY temporary network code?

### ISSUE THREAD

**Issue by WayneCrawford** [FDSN/source-identifiers#24]
For seagoing missions, at least, we are often asked to provide the data to the chief scientist before it is ready for archiving properly.  This data needs to be clearly marked as "preliminary" and one of the simplest ways is to give it a "nonsense" network code.  But, as far as I know, there isn't a network code that is disallowed by FDSN but which will be properly processed by seismological software.

Would it make sense to define a "never-FDSN" code that could be used for this?  With the new network code definition, this could simply be using "0000" as the year.  But it might be good to write this somewhere so that people futzing with preliminary data (or synthetic data that will never go on FDSN) can use a code that clearly shows it is non-FDSN.

**Comment by crotwell** [comment]
The `XX` network code could be used for that I think, as it is defined to be:
```
XX – used for an experimental temporary or permanent network. Data collected with this Network Code should never be distributed.
```
defined in an old [IRIS newsletter](https://ds.iris.edu/ds/newsletter/vol1/no1/1/specification-of-seismograms-the-location-identifier/). I looked on fdsn.org but did not find this.

@chad-iris do you think it would be good to actually register the `XX` network code within the FDSN system just to provide documentation that it shouldn't be used for distributed data? I don't think there is any danger of anyone trying to request it, but there is nothing I could find on the fdsn website that defines it.

Also, given the temp network rules, I think we have effectively said that network codes that look like XX2013 would never be issued, so I guess that is also an option.

On the other hand, I am not sure using the network code to mark data as "preliminary" is really the best way to do that. Seems like it would be better to use the actual network code, and then mark the metadata and waveforms using some other attribute? Miniseed has the "data quality" byte that might be more applicable. The data quality byte can be set to `R` to mean:
```
R — Raw Waveform Data with no Quality Control
```

Not sure if that is really what you need. Is this a "not corrected for known errors (like time)" or a "top secret, restricted, your eyes only" concept, or both?

**Comment by chad-earthscope** [comment]
Yes, the XX network is what IRIS recommends for a placeholder, for when the code is not yet known or the data is not going to be shared, or any reason that a real code is not desired but one is needed for the formats.

> @chad-iris do you think it would be good to actually register the XX

I do and have raised the issue with the folks responsible for registering codes.

This should have been documented by the FDSN all along IMHO, and so I think we should codify it as a recommendation for use where needed in this spec if we can find the right place.  Any recommendations on where?

**Comment by crotwell** [comment]
How about at the bottom of the Network section, just above the Location code usage heading. Maybe:

Placeholder Network Codes

The 2 character network code `XX` has been reserved for an experimental temporary or permanent network. 
It may be used as a placeholder for seismic data that does not yet, or never will have a fdsn issued network code. Data collected with this Network Code should never be distributed.

In addition, following the transitional mapping of previously allocated temporary network codes convention, all network codes of the form
```
XX<4-digit start year>
```
are also reserved for experimental use. Data with network codes of this form should also never be distributed.

**Comment by crotwell** [comment]
Also probably should have the `SS` network?

The 2 character network code `SS` may be used by any institution running a Single Station, but this station should be registered with the FDSN. Care must be given to insure that the Station Identifier (described below) is not identical to another station using the SS network code. Data collected with this Network Code may be distributed.

**Comment by chad-earthscope** [comment]
OK, I've added this into a section "Special network codes", under a new "Network codes" section.

Somewhat related, we've added so much to the definition of identifiers page that it was getting too big.  So I broke out the network and location code descriptions.  You'll have to wait until readthedocs caching system lets the new renders out the door (not sure what's up with that), maybe clear your browser cache.
